### PR TITLE
Update `Gameplay/Game modifier/Perfect`

### PR DESCRIPTION
**Pull request by heipizhu4**
Just discovered this 'feature' and decided to add this to the wiki
4948aec655f119ac

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x]  _(translations only)_ The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Gameplay/Game_modifier/Perfect/en.md [hunk 1]
```diff
@@ -40,8 +40,8 @@ Any of the following acts **will not cause** the Perfect mod to restart the beat
 - Obtaining a `300`
 - Completing a spinner
 - Obtaining a Rainbow 300/MAX score ([osu!mania](/wiki/Game_mode/osu!mania))
-- Obtaining a 300 score value (osu!mania)
-
+- Obtaining a 300 score value ([osu!mania](/wiki/Game_mode/osu!mania))
+- Obtaining a 50/100/200 score value on the head/tail of a long note in [ScoreV2](/wiki/en/Gameplay/Game_modifier/ScoreV2) ([osu!mania](/wiki/Game_mode/osu!mania))
```
Comment (agatemosu) [inline wiki/Gameplay/Game_modifier/Perfect/en.md:44]:
```suggestion
- Obtaining a 50/100/200 score value on the head/tail of a long note in [ScoreV2](/wiki/en/Gameplay/Game_modifier/ScoreV2) ([osu!mania](/wiki/Game_mode/osu!mania))

```

**Review by agatemosu — CHANGES_REQUESTED** [review]

**Review by ChickenGoldSS — APPROVED** [review]

File: wiki/Gameplay/Game_modifier/Perfect/en.md [hunk 2]
```diff
@@ -40,7 +40,8 @@ Any of the following acts **will not cause** the Perfect mod to restart the beat
 - Obtaining a `300`
 - Completing a spinner
 - Obtaining a Rainbow 300/MAX score ([osu!mania](/wiki/Game_mode/osu!mania))
-- Obtaining a 300 score value (osu!mania)
+- Obtaining a 300 score value ([osu!mania](/wiki/Game_mode/osu!mania))
```
Comment (Walavouchey) [inline wiki/Gameplay/Game_modifier/Perfect/en.md:43]: isn't this covered by the first item in this list? (line 40)

Comment (heipizhu4) [reply wiki/Gameplay/Game_modifier/Perfect/en.md]: Fixed, thanks!

File: wiki/Gameplay/Game_modifier/Perfect/en.md [hunk 3]
```diff
@@ -40,7 +40,8 @@ Any of the following acts **will not cause** the Perfect mod to restart the beat
 - Obtaining a `300`
 - Completing a spinner
 - Obtaining a Rainbow 300/MAX score ([osu!mania](/wiki/Game_mode/osu!mania))
-- Obtaining a 300 score value (osu!mania)
+- Obtaining a MAX in ([osu!mania](/wiki/Game_mode/osu!mania))
```
Comment (agatemosu) [inline wiki/Gameplay/Game_modifier/Perfect/en.md:43]: Now it's the same the one above this

**Review by agatemosu — COMMENTED** [review]
I think that now it should be put like this (in line 32 and 29 also change it to "OK"/"MEH"/"GREAT")

File: wiki/Gameplay/Game_modifier/Perfect/en.md [hunk 4]
```diff
@@ -37,10 +37,10 @@ Any of the following acts **will cause** the Perfect mod to restart the beatmap:
 
 Any of the following acts **will not cause** the Perfect mod to restart the beatmap:
 
-- Obtaining a `300`
+- Obtaining a GREAT
```
Comment (TicClick) [inline wiki/Gameplay/Game_modifier/Perfect/en.md:40]: unless I'm missing something, this is still called a 300 in all the other game modes

Comment (TicClick) [reply wiki/Gameplay/Game_modifier/Perfect/en.md:40]: [yeah](https://github.com/ppy/osu-wiki/pull/8245#discussion_r1002804800)

**Review by TicClick — APPROVED** [review]

File: wiki/Gameplay/Game_modifier/Perfect/zh.md [hunk 5]
```diff
@@ -42,7 +42,8 @@ tags:
 - 获得 `300`
 - 完成转盘
 - 获得彩虹 300/MAX 得分（[osu!mania](/wiki/Game_mode/osu!mania)）
-- 获得 300 分（osu!mania）
+- 获得 300 分([osu!mania](/wiki/Game_mode/osu!mania))
+- 在 [ScoreV2](/wiki/Gameplay/Game_modifier/ScoreV2) 下，长条的头/尾获得200/100/50的判定([osu!mania](/wiki/Game_mode/osu!mania))
```
[the remark this answers is not in the document]
Comment (heipizhu4) [reply wiki/Gameplay/Game_modifier/Perfect/zh.md]: Fixed.

**Comment by heipizhu4** [comment]
> I suggest changing line 29 to:
> 
> ```
> The **Perfect** mod is a [game modifier](/wiki/Gameplay/Game_modifier) that aims to aid players who are going for an SS grade by automatically failing and retrying [beatmaps](/wiki/Beatmap) if the player misses a note or obtains any judgement on a [hit object](/wiki/Gameplay/Hit_object) other than GREAT.
> ```
> 
> And it remains to change line 34

Fixed.

File: wiki/Gameplay/Game_modifier/Perfect/en.md [hunk 6]
```diff
@@ -24,23 +24,21 @@ tags:
 - Compatible game modes: ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
 - Variant: [Sudden Death](/wiki/Gameplay/Game_modifier/Sudden_Death)
 
-## Description
-
-The **Perfect** mod is a [game modifier](/wiki/Gameplay/Game_modifier) that aims to aid players who are going for an SS grade by automatically failing and retrying [beatmaps](/wiki/Beatmap) if the player misses a note or obtains any score on a [hit object](/wiki/Gameplay/Hit_object) other than 300.
+## DescriptionThe **Perfect** mod is a [game modifier](/wiki/Gameplay/Game_modifier) that aims to aid players who are going for an SS grade by automatically failing and retrying [beatmaps](/wiki/Beatmap) if the player misses a note or obtains any judgement on a [hit object](/wiki/Gameplay/Hit_object) other than GREAT.
```
Comment (agatemosu) [inline wiki/Gameplay/Game_modifier/Perfect/en.md:27]:
```suggestion
The **Perfect** mod is a [game modifier](/wiki/Gameplay/Game_modifier) that aims to aid players who are going for an SS grade by automatically failing and retrying [beatmaps](/wiki/Beatmap) if the player misses a note or obtains any judgement on a [hit object](/wiki/Gameplay/Hit_object) other than GREAT.
```

File: wiki/Gameplay/Game_modifier/Perfect/zh.md [hunk 7]
```diff
@@ -28,21 +28,21 @@ tags:
...
 
 - 漏音符
-- 获得 `100` 或 `50`
+- 获得 `GOOD` 或 `MEH`
 - 未能完成转盘
 - 断滑条
 
 下列情况**不会导致**此模组重新开始谱面：
 
 - 获得 `300`
```
Comment (TicClick) [inline wiki/Gameplay/Game_modifier/Perfect/zh.md:42]: shouldn't this be `GREAT` instead, for consistency?

Comment (heipizhu4) [reply wiki/Gameplay/Game_modifier/Perfect/zh.md:42]: Nope, when you get a GREAT in lazer, you got a 300.

Comment (agatemosu) [reply wiki/Gameplay/Game_modifier/Perfect/zh.md:42]: image

File: wiki/Gameplay/Game_modifier/Perfect/en.md [hunk 8]
```diff
@@ -26,21 +26,21 @@ tags:
...
 ## Description
 
-The **Perfect** mod is a [game modifier](/wiki/Gameplay/Game_modifier) that aims to aid players who are going for an SS grade by automatically failing and retrying [beatmaps](/wiki/Beatmap) if the player misses a note or obtains any score on a [hit object](/wiki/Gameplay/Hit_object) other than 300.
+The **Perfect** mod is a [game modifier](/wiki/Gameplay/Game_modifier) that aims to aid players who are going for an SS grade by automatically failing and retrying [beatmaps](/wiki/Beatmap) if the player misses a note or obtains any judgement on a [hit object](/wiki/Gameplay/Hit_object) other than GREAT.
 
 Any of the following acts **will cause** the Perfect mod to restart the beatmap:
 
 - Missing a note
-- Obtaining a `100` or `50`
+- Obtaining a `GOOD` or `MEH`
```
Comment (agatemosu) [inline wiki/Gameplay/Game_modifier/Perfect/en.md:34]:
```suggestion
- Obtaining a GOOD or MEH
```

File: wiki/Gameplay/Game_modifier/Perfect/zh.md [hunk 9]
```diff
@@ -28,21 +28,21 @@ tags:
...
 ## 说明
 
-**Perfect** [游戏模组](/wiki/Gameplay/Game_modifier)通过在漏掉音符、[打击物件](/wiki/Gameplay/Hit_object)获得 300 以外的得分等等情况下，自动认定失败、重新开始[谱面](/wiki/Beatmap)，旨在帮助想要获得 SS 等级的玩家。
+**Perfect** [游戏模组](/wiki/Gameplay/Game_modifier)通过在漏掉音符、[打击物件](/wiki/Gameplay/Hit_object)获得 PERFECT 以外的得分等等情况下，自动认定失败、重新开始[谱面](/wiki/Beatmap)，旨在帮助想要获得 SS 等级的玩家。
 
 下列任一情况**会导致**此模组重新开始谱面：
 
 - 漏音符
-- 获得 `100` 或 `50`
+- 获得 `GOOD` 或 `MEH`
```
Comment (agatemosu) [inline wiki/Gameplay/Game_modifier/Perfect/zh.md:36]:
```suggestion
- 获得 GOOD 或 MEH
```

File: wiki/Gameplay/Game_modifier/Perfect/zh.md (hunk 7 above)
Comment (agatemosu) [inline wiki/Gameplay/Game_modifier/Perfect/zh.md:42]:
```suggestion
- 获得 GREAT
```

File: wiki/Gameplay/Game_modifier/Perfect/zh.md [hunk 10]
```diff
@@ -28,21 +28,21 @@ tags:
...
 - 未能完成转盘
 - 断滑条
 
 下列情况**不会导致**此模组重新开始谱面：
 
 - 获得 `300`
 - 完成转盘
-- 获得彩虹 300/MAX 得分（[osu!mania](/wiki/Game_mode/osu!mania)）
-- 获得 300 分（osu!mania）
+- 获得 MAX 判定（[osu!mania](/wiki/Game_mode/osu!mania)）
```
Comment (agatemosu) [inline wiki/Gameplay/Game_modifier/Perfect/zh.md:44]:
```suggestion
- 获得 PERFECT 判定（[osu!mania](/wiki/Game_mode/osu!mania)）
```

**Review by agatemosu — CHANGES_REQUESTED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**